src/styles/variables.scss

```
@@ -0,0 +1,4 @@
+$collor-button: #00acdc;
+$important-text-collor: #060b35;
+$not-important-text: #616070;
+$color-for-header: #00acdc;
```

Why did you create two variables with the same values?
```suggestion
$color-blue: #00acdc;
$important-text-color: #060b35;
$not-important-text: #616070;
```